### PR TITLE
Fixes #514 Require inputing name for `playlist make <name>` command

### LINKED ISSUE #514: [Bug Report] Playlist with empty names can be created
**Describe the bug**
When using [prefix]playlist make, a playlist is created with the name " ", and a '.txt' file is created in the Playlists folder.

**To Reproduce**
1. Execute the `playlist make` command with no arguments passed

**Expected behavior**
The bot shouldn't create any playlist and should throw an error message with something like "Please enter a playlist name"

**Screenshots**
File with an empty name:
image
"Successfully created playlist ``!"
image

**Debug Info**
<details>
  <summary>Debug info</summary>

```
System Properties:
  java.version = 1.8.0_265
  java.vm.name = OpenJDK 64-Bit Server VM
  java.vm.specification.version = 1.8
  java.runtime.name = OpenJDK Runtime Environment
  java.runtime.version = 1.8.0_265-8u265-b01-0+deb9u1-b01
  java.specification.version = 1.8
  os.arch = amd64
  os.name = Linux

JMusicBot Information:
  Version = 0.2.10
  Owner = 278230133176008704
  Prefix = //
  AltPrefix = ..
  MaxSeconds = 0
  NPImages = false
  SongInStatus = false
  StayInChannel = false
  UseEval = false
  UpdateAlerts = true

Dependency Information:
  JDA Version = 3.8.3_464
  JDA-Utilities Version = 2.1.5
  Lavaplayer Version = 1.3.50

Runtime Information:
  Total Memory = 648
  Used Memory = 206

Discord Information:
  ID = 512030715332919348
  Guilds = 100
  Users = 101485
```

</details>

**Additional context**
The playlist can be successfully deleted using `playlist delete` without arguments.

### PR DESCRIPTION
**Pull request by all-laugh**
playlist make command without a name will now not make an empty playlist without a playlist name.

### This pull request...
  - [x] Fixes a bug
  - [ ] Introduces a new feature
  - [ ] Improves an existing feature
  - [ ] Boosts code quality or performance

### Description
Checks whether we have obtained a name from the user for generating a new, empty playlist. If not, we ask for a name.

### Purpose
Fix bug #514 where empty playlists with no name can be created

### Relevant Issue(s)
This PR closes issue #514

### CODE REVIEW

File: src/main/java/com/jagrosh/jmusicbot/commands/owner/PlaylistCmd.java [hunk 1]
```diff
@@ -72,7 +72,16 @@ public MakelistCmd()
         protected void execute(CommandEvent event) 
         {
             String pname = event.getArgs().replaceAll("\\s+", "_");
-            if(bot.getPlaylistLoader().getPlaylist(pname)==null)
+            if(pname == null || pname.isEmpty()) 
+            {
+                event.reply(event.getClient().getError() + " Please provide a name for playlist!");
```
Comment (MichailiK) [inline src/main/java/com/jagrosh/jmusicbot/commands/owner/PlaylistCmd.java:77]: Old code may still use this, but you should use `event.replyError` instead. Additionally, the grammar should be slightly adjusted:
```suggestion
                event.replyError("Please provide a name for the playlist!");
```

File: src/main/java/com/jagrosh/jmusicbot/commands/owner/PlaylistCmd.java [hunk 2]
```diff
@@ -72,7 +72,16 @@ public MakelistCmd()
...
+                
+                StringBuilder builder = new StringBuilder(event.getClient().getWarning() + " Playlist Make Command:\n");
+                builder.append("\n`").append(event.getClient().getPrefix()).append(name)
+                        .append(" <name>` - creates a new, empty playlist with given name");
```
Comment (MichailiK) [inline src/main/java/com/jagrosh/jmusicbot/commands/owner/PlaylistCmd.java:81]: I personally believe that this is too excessive.

**Review by MichailiK — CHANGES_REQUESTED** [review]

File: src/main/java/com/jagrosh/jmusicbot/commands/owner/PlaylistCmd.java (hunk 2 above)
Comment (all-laugh) [reply src/main/java/com/jagrosh/jmusicbot/commands/owner/PlaylistCmd.java:81]: I think so too :) Just wanted to match with the hints that were given if I just call the `playlist` command with no arguments. I'll get rid of it.

**Comment by all-laugh** [comment]
@MichailiK the requested changes have been committed. Let me know your thoughts!

**Review by MichailiK — APPROVED** [review]

**Comment by jagrosh** [comment]
As an aside, the args will never be null, so the null check here is unnecessary, but it also really doesn't hurt anything either.

**Pull request merged**